sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -1361,6 +1355,37 @@ class HiveDDLSuite
     }
   }
 
+  test("create hive serde table with DataFrameWriter.saveAsTable with partitionBy") {
```

Comment:
nit: `create partitioned hive serde table as select`, and we should test both `DataFrameWriter.saveAsTable` and `CREATE TABLE ... PARTITIONED BY (...) AS SELECT`